R/environment.r

```diff
@@ -44,3 +44,23 @@ init_shadowenv <- function() {
         stop("'edit()' not yet supported in the Jupyter R kernel")
     })
 }
+
+
+init_cran_repo <- function() {
+    local({
+        r <- getOption('repos');
+        is_configured_by_user <- !is.null(r) && identical(r, '@CRAN@')
```

Comment:
the null check is superfluous.

please just copy&paste [this version](https://github.com/IRkernel/IRkernel/pull/339#issuecomment-220296012)